### ISSUE TITLE
[@material/react-switch] Missing aria-checked attribute in input element

### ISSUE THREAD

**Issue by heyitsjhu** [material-components/material-components-web-react#996]
Hi,

When I render the Switch component in Storybook with the A11Y addon, I get the following accessibility violation: "Required ARIA attribute not present: aria-checked." 

![image](https://user-images.githubusercontent.com/14210200/63875558-8d932500-c978-11e9-8912-182f40d1d3f1.png)

![image](https://user-images.githubusercontent.com/14210200/63875578-997ee700-c978-11e9-98d2-d44797775f3f.png)

I tried adding `aria-checked` and `ariaChecked` as Switch component props to see if it gets passed down directly, but it doesn't appear so. However, when I update the HTML directly and add in the attribute, the error in storybook goes away.

![image](https://user-images.githubusercontent.com/14210200/63875782-10b47b00-c979-11e9-851a-1883e354a91f.png)

Is this something we can add to the component or is there a reason why it's omitted from the element? I'm also happy to take a stab at creating a pull request if we decided to add the attribute. :)

**Comment by stefcot** [comment]
Just try 
```
          inputProps={{
            'aria-checked': toggled || undefined,
          }}```
